tools/terrabyte_ogc_api_processes/terrabyte_ogc_api_process_wrapper.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="terrabyte_ogc_api_processes" name="Terrabyte OGC API Processes" version="0.1.0" profile="22.05">
+	<description>Water quality EO Application Package</description>
+	<requirements>
+		<requirement type="package" version="4.3.1">r-base</requirement>
+		<requirement type="package" version="1.20.4">r-getopt</requirement>
+		<requirement type="package" version="0.2.3">r-httr2</requirement>
+		<requirement type="package" version="1.8.7">r-jsonlite</requirement>
+	</requirements>
+	<command detect_errors="exit_code"><![CDATA[
+	Rscript '$__tool_directory__/terrabyte_ogc_api_process_wrapper.R'
+		--outputData '$output_data'
+]]></command>
+	<configfiles>
+		<inputs name="inputs" filename="inputs.json" data_style="paths"/>
+	</configfiles>
+	<inputs>
+		<param name="cookie" label="Cookie" optional="false" value="" type="text" help="See 'Help' for for more information regarding the cookie."/>
```

Comment:
There is no help about this.

Is there no other way. This seems to be a general problem which we need to address.

Reply:
Cookies/Tokens and in general authentication are a problem in our project. I hope this will be addressed in the second phase of the project...

Reply:
But realistically, how likely is it that someone can paste a cookie in here? And even worse this cookie is saved in the DB - if you share your history someone else has access to it.

Reply:
The cookies expire after a few minutes. 
I am also not very happy with this cookie issue, but this is nothing we can solve until the end of the first project phase, which will be in six weeks. They are also aware of this issue and I hope that this demonstrator will provide some motivation to work in it during the next months.